portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <!-- Imported Libraries: Google's Material Design Library and Google's Fonts Library -->
+    <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+    <title>MM Portfolio</title>
+    <link rel="stylesheet" href="blog_style.css">
+    <script src="blog_script.js"></script>
+  </head>
+  <body>
+    <div class="mdc-touch-target-wrapper" style="display: block;">
+	  <button class="mdc-button mdc-button--outlined" onclick="return_home()" style="color:white;">  
+	    <span class="mdc-button__ripple">Return</span>
+	  </button>
+    </div>
+    <div id="my-profile">
```

Comment:
nit - update the id to reflect what's actually in the `div`

Reply:
Noted! Thanks!